src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -391,7 +391,8 @@ module Make (Shifted_value : Shifted_value.S) (Sc : Scalars.S) = struct
         ; beta
         ; gamma
         ; zeta
-        ; zeta_to_domain_size = env.zeta_to_n_minus_1 + F.one
+        ; zeta_to_domain_size =
+            env.zeta_to_n_minus_1 + F.one (* [\zeta^n - 1] + 1 = \zeta^n. *)
```

Comment:
Is this a useful comment? Hopefully it's obvious from context, but it would be good to rename the variable if not, rather than commenting

Reply:
No, it is not useful :smile:

Reply:
Removed.